Automattic/wp-calypso
# Changing the re-direct URL of the resetting password page for Gravatar

Related to p9lV3a-5P8-p2

## Proposed Changes

* Changing the re-direct URL of the resetting password page for Gravatar

## Testing Instructions



* Switch to the branch and run it locally (check [the doc](https://github.com/Automattic/wp-calypso#getting-started) to setup your dev environment)
* Log out your WP account
* Go to the log-in page by [this URL](http://calypso.localhost:3000/log-in?client_id=1854&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3Dad4b462bedc61ff0ee2b72b16252632bcf4f4ec2e9aaab25a169e49d565e87bf%26redirect_uri%3Dhttps%253A%252F%252Fen.gravatar.com%252Fconnect%252F%253Faction%253Drequest_access_token%26from-calypso%3D1)
* Click the "Lost your password" link
* Enter your account or email and then click the "Get New Password" button. You will be redirected to the login page of Gravatar as below:

| Before | After |
| - | - |
| <img width="1728" alt="截圖 2023-04-28 下午2 25 05" src="https://user-images.githubusercontent.com/21308003/235074905-aeec1508-8c54-46b4-a8d0-5787576d47cb.png"><img width="1728" alt="截圖 2023-04-28 下午2 26 09" src="https://user-images.githubusercontent.com/21308003/235074948-eded60ce-02f0-4192-81e0-9f05dbd2a3f3.png"> | <img width="1728" alt="截圖 2023-04-28 下午2 25 05" src="https://user-images.githubusercontent.com/21308003/235074905-aeec1508-8c54-46b4-a8d0-5787576d47cb.png"><img width="1728" alt="截圖 2023-04-28 下午2 25 33" src="https://user-images.githubusercontent.com/21308003/235075038-5326c3ca-ac07-41f8-9e7e-e1af619e067f.png"> |

## Pre-merge Checklist
